content/authoring/guidelines/index.md

```diff
@@ -5,8 +5,30 @@ prev: /authoring/
 next: /authoring/guidelines/description/
 ---
 
-# Authoring Guidelines
+# Codewars Content Quality Guidelines
 
-_This page is a stub._
+## Overview
 
-General guidelines and overview.
+This set of pages contains guidelines for kata authors and translators to help them create kata, translations and kumite of sufficient quality, so the experience of working with them will be as good as possible for all Codewars users.
+
+The guidelines should be used by kata authors, translators and reviewers to verify whether new content about to be introduced to Codewars is of sufficient quality. Conformance with them should be verified before a kata, translation, or kumite is published by its author and approved by a reviewer.
+
+All content provided by Codewars is created by the community, and for the community. No one else but the community itself can help the most to keep the quality good and constantly improve it. All Codewars users are encouraged to provide suggestions for improvements, report encountered issues, and fix them.
+
+As the guidelines were introduced after a lot of kata had already been created, it's obvious that they may not be respected by existing kata and translations to the required extent. In such cases, beta kata and pending translations should be updated to improve their quality as much as possible before they get approved. Existing kata, on the other hand, can be approached in a more relaxed way. Obvious bugs and severe issues should be still fixed, but otherwise, guidelines can be applied gradually as the kata receives new translations, or as issues are raised and then fixed.
+
+Some of these guidelines do not apply to every type of content. There are code golf kata, puzzle kata, hackmes, debugging kata, and many other problems which are designed on the grounds of ignoring some of the guidelines. They should be used where applicable, and can be skipped when they are not. But this should not be treated as an excuse for taking shortcuts or skipping the necessary work: the fact that some of the guidelines are not relevant should be clearly visible from the task.
+
+:::warning Important note
+Failure to comply with the guidelines should be considered an issue to be addressed and reported as such. In case of severe violations, the affected kata or translation may be retired, moved back to beta or rejected.
```

Comment:
```suggestion
Failure to comply with the guidelines should be considered an issue to be addressed and reported. In case of severe violations, the affected kata or translation may be retired, moved back to beta or rejected.
```